cmd/tailscaled/tailscaled.go

```diff
@@ -109,7 +115,8 @@ func main() {
 		e, err = wgengine.NewUserspaceEngine(logf, *tunname, *listenport)
 	}
 	if err != nil {
-		log.Fatalf("wgengine.New: %v", err)
+		logf("wgengine.New: %v", err)
```

Comment:
log.Fatalf exits with a non-zero exit code. I think this change breaks that, so we'll exit 0 now. Not sure if we care?...

Reply:
Oops, thanks for catching this. We actually do, since that controls whether `systemd` will run ExecStopPost. That makes is somewhat trickier. I guess I'll refactor the whole function...